ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ip-hardware.j2

```diff
@@ -0,0 +1,7 @@
+{# eos - IP Hardware #}
+{% if ip_hardware is arista.avd.defined %}
+!
+{%   if ip_hardware.fib.optimize.prefix_profile is arista.avd.defined %}
+ip hardware fib optimize prefixes profile {{ ip_hardware.fib.optimize.prefix_profile }}
```

Comment:
I think we should change the data model to match the EOS syntax:
```
smv031(config)#ip hardware fib optimize ?
  disable-vrf    Disable fib optimizations in Vrf
  prefix-length  Length of the prefix in bits
  prefixes       Optimize prefixes
  vrf            Optimize IPv4 routes for specific VRF(s)
  <cr>
```
So it should be:
```
ip_hardware:
  fib:
    optimize:
      prefixes:
        profile: < internet | urpf-internet >